scripts/oracle/price_feeder.py

```diff
@@ -46,51 +39,17 @@ def init_val_addr(self):
     def get_current_vote_period(self):
         res = requests.get("{node}/blockchain".format(node=self.node))
         body = res.json()
-        return int(body["result"]["last_height"]) // 10
+        return int(body["result"]["last_height"])
```

Comment:
yeah it would allow us to vote more frequently, although come to think of it, the atlantic-1 voting period is still 10 so this doesn;t really accomplish anything, I'll change this back